common/supplemental/coverageLevels.xml

```diff
@@ -904,23 +906,17 @@ For terms of use, see http://www.unicode.org/copyright.html
-		<coverageLevel inLanguage="%personNameLanguages" value="modern" match="personNames/personName[@length='%anyAttribute'][@usage='%anyAttribute'][@order='%anyAttribute']/namePattern[@_q='%anyAttribute']"/>
-		<coverageLevel inLanguage="%personNameLanguages" value="modern" match="personNames/personName[@length='%anyAttribute'][@usage='%anyAttribute']/namePattern[@_q='%anyAttribute']"/>
-		<coverageLevel inLanguage="%personNameLanguages" value="modern" match="personNames/personName[@length='%anyAttribute'][@style='%anyAttribute'][@order='%anyAttribute']/namePattern[@_q='%anyAttribute']"/>
-		<coverageLevel inLanguage="%personNameLanguages" value="modern" match="personNames/personName[@length='%anyAttribute'][@style='%anyAttribute']/namePattern[@_q='%anyAttribute']"/>
-		<coverageLevel inLanguage="%personNameLanguages" value="modern" match="personNames/personName[@length='%anyAttribute'][@order='%anyAttribute']/namePattern[@_q='%anyAttribute']"/>
-		<coverageLevel inLanguage="%personNameLanguages" value="modern" match="personNames/personName[@length='%anyAttribute']/namePattern[@_q='%anyAttribute']"/>
-		<coverageLevel inLanguage="%personNameLanguages" value="modern" match="personNames/personName[@usage='%anyAttribute'][@style='%anyAttribute'][@order='%anyAttribute']/namePattern[@_q='%anyAttribute']"/>
-		<coverageLevel inLanguage="%personNameLanguages" value="modern" match="personNames/personName[@usage='%anyAttribute'][@style='%anyAttribute']/namePattern[@_q='%anyAttribute']"/>
-		<coverageLevel inLanguage="%personNameLanguages" value="modern" match="personNames/personName[@usage='%anyAttribute'][@order='%anyAttribute']/namePattern[@_q='%anyAttribute']"/>
-		<coverageLevel inLanguage="%personNameLanguages" value="modern" match="personNames/personName[@usage='%anyAttribute']/namePattern[@_q='%anyAttribute']"/>
-		<coverageLevel inLanguage="%personNameLanguages" value="modern" match="personNames/personName[@style='%anyAttribute'][@order='%anyAttribute']/namePattern[@_q='%anyAttribute']"/>
-		<coverageLevel inLanguage="%personNameLanguages" value="modern" match="personNames/personName[@style='%anyAttribute']/namePattern[@_q='%anyAttribute']"/>
-		<coverageLevel inLanguage="%personNameLanguages" value="modern" match="personNames/personName[@order='%anyAttribute']/namePattern[@_q='%anyAttribute']"/>
-		<coverageLevel inLanguage="%personNameLanguages" value="modern" match="personNames/personName/namePattern[@_q='%anyAttribute']"/>
+		<coverageLevel inLanguage="%personNameLanguages" value="modern" match="personNames/nameOrder[@nameLocales='%anyAttribute']"/>
+		<coverageLevel inLanguage="%personNameLanguages" value="modern" match="personNames/personName[@length='%personNameNonMonograms'][@usage='addressing'][@style='%anyAttribute'][@order='%anyAttribute']/namePattern[@alt='%anyAttribute']"/>
```

Comment:
We have two situations:
In our source, we will have fully qualified. So just lines 911 and 912.

In the production data, we would have to have all the combinations

- length, usage, style, order
- length, usage, style
- length, usage, order
- length, style, order
- usage, style, order
- length, usage
- length, style
- length, order
- usage, style
- usage, order
- style, order
- length
- usage
- style
- order

EXCEPT: we should never have to apply coverage to the production data.

Reply:
So I think just lines 911 and 912 are good enough.

Reply:
See my comments in the PR overview and the e-mail I sent. I think there is no need to have fully qualified in the source for attributes that are irrelevant, like usage for monograms, or order for usage=sorting. What would be the point? The regular cldr data should b fully fleshed out *except* for those cases (i.e. for monograms we don't need any entries with usage attribute). That is the way I entered the data, so that is the way I have coverage setup (and PathHeader, etc.).

Reply:
Ok, makes sense.